test/parallel/test-fs-error-messages.js

```diff
@@ -21,14 +21,15 @@
 
 'use strict';
 const common = require('../common');
+const fixtures = require('../common/fixtures');
 const assert = require('assert');
 const path = require('path');
```

Comment:
If I am not mistaken, `path` is not actually used anymore, so this line should be removed.